### ISSUE TITLE
Add mailing list instructions to README

### ISSUE THREAD

**Issue by berezovskyi** [4store/4store#154]
@er5678ihgfrt5y6ftgtyr raised a question of using the mailing list without a Google acct.

I found the following instructions: http://blog.neowork.com/how-to-subscribe-to-a-google-group-without-a-google-account and the mailing list page: https://groups.google.com/forum/#!aboutgroup/4store-support

README should be updated.

@@er5678ihgfrt5y6ftgtyr can you give it a try and maybe update the README, please?

**Comment by berezovskyi** [comment]
I was able to sign up using email and send a reply to https://groups.google.com/forum/#!topic/4store-support/Q-w88iKWFmQ but I can't see it yet. Perhaps, @swh needs to approve it first.

**Comment by swh** [comment]
Yeah, we turned on first-post approval, to combat spam. It's approved now.

**Comment by berezovskyi** [comment]
*Re: 4store.org* put my reply under the wrong topic, so I removed it and sent another email with the * 4store.org* subject. Let's see if it lands in the right place.

**Comment by berezovskyi** [comment]
So, interacting with the mailing list without a Google acct seems to work but replying to the old topic from the email client seems to be next to impossible: https://groups.google.com/forum/#!forum/4store-support

**Comment by swh** [comment]
Well, that's normal for mailing lists - without the message headers from the old thread, the threading goes all wonky.
